math/lapack/Portfile

```diff
@@ -2,21 +2,28 @@
 
 PortSystem          1.0
 PortGroup           github 1.0
+PortGroup           compilers 1.0
+PortGroup           cmake 1.0
 
-github.setup        Reference-LAPACK lapack 3.11.0 v
+github.setup        Reference-LAPACK lapack 3.12.0 v
 if {${subport} eq ${name}} {
-    PortGroup           compilers 1.0
     compilers.choose    cc fc f77 f90
     compilers.setup     require_fortran -clang
     compilers.add_gcc_rpath_support no
 
-    PortGroup           cmake 1.0
     cmake.out_of_source yes
```

Comment:
this can/should be removed as it is the default in the most recent `cmake` PortGroup version

Reply:
Done.